### ISSUE TITLE
Docker push to hub fails

### ISSUE THREAD

**Issue by dsampath** [docker/hub-beta-feedback#273]
1. Checked the admin privileges of the users - OK (user has write permissions) 
2. Checked the login creds on the machine locally - OK (able to pull from private repos indicating that I am logged in)

`docker push` managed to push a few layers before this message appears. This changes with each push, sometimes it is instantaneous, sometimes it's after pushing a few layers. 

```
Error parsing HTTP response: invalid character '<' looking for beginning of value: "<html><body><h1>403 Forbidden</h1>\nRequest forbidden by administrative rules.\n</body></html>\n\n"`
```

**Comment by xavi-** [comment]
Seeing this too?  Any response from dockerhub?

**Comment by dmp42** [comment]
@dsampath @xavi- if you see this, then you are using docker 1.8.0 - which shipped with a bug on push.

We immediately shipped docker 1.8.1 to address the issue, and are now blocking 1.8.0 from pushing to the hub to prevent any damage.

The solution is simple: update to docker 1.8.1.

Let me know if that helps.

**Comment by xavi-** [comment]
Definitely does.  We finally figured it out earlier this evening.  Thanks.
